Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM python:3 as base
+
+#ENV LIBRARY_PATH=/lib:/usr/lib
+
+RUN curl -sSL https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py | python
+
+WORKDIR /todo_app
+
+COPY . /todo_app
+#copy dir
+ENV PATH="${PATH}:/root/.poetry/bin"
+RUN poetry install
+
+#EXPOSE 80
+#install poetry
+FROM base as development 
+ENTRYPOINT  poetry run flask run --host 0.0.0.0:80
+
+FROM base as production
+RUN poetry add gunicorn
```

Comment:
If you run this command from the terminal _outside_ the container then Gunicorn will be registered as a project dependency and it will be installed automatically when future people/images use `poetry install`, which would mean:
* We can avoid running it fresh every time here
* More importantly - we have a more stable version of Gunicorn in use, rather than potentially varying each time we build the image